binary_transparency/firmware/common/key_info.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC. All Rights Reserved.
```

Comment:
This package might be better in `internal/crypto` (or perhaps `/internal/sig`?)
(we're hiding implementation under internal unless we explicitly choose to expose it)